src/status_im/contexts/wallet/send/from/view.cljs

```diff
@@ -23,10 +24,17 @@
 
 (defn- render-fn
   [item]
-  (let [network-details (rf/sub [:wallet/network-details])]
+  (let [network-details     (rf/sub [:wallet/network-details])
+        network-short-names (map #(network-utils/network->short-name %)
+                                 (:network-preferences-names item))
```

Comment:
We don't need to wrap the `network-utils/network->short-name` in another funciton

Reply:
Done